.gitignore

```diff
@@ -331,7 +335,7 @@ ASALocalRun/
 
 # Visual Studio Code
 .vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
+.vscode/settings.json
```

Comment:
I believe these lines are now redundant due to the line above ignoring `.vscode/*`